configure

```diff
@@ -2537,6 +2548,17 @@ if test -n "$_host"; then
 			_seq_midi=no
 			_port_mk="backends/platform/dingux/dingux.mk"
 			;;
+		raspberrypi)
+			# For proper cross-compilation, we assume that the cross-building environment provides 
```

Comment:
For cross compilation we usually pick up something like host-sdl-config which should have the appropriate prefix, which does not necessarily need to concide with $RPI_ROOT/usr. Please check that this works as expected.